migration-0.45/dashboards.go

```diff
@@ -0,0 +1,130 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"time"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/mattn/go-sqlite3"
+)
+
+var (
+	db *sqlx.DB
+)
+
+type Dashboard struct {
+	Id        int       `json:"id" db:"id"`
+	Uuid      string    `json:"uuid" db:"uuid"`
+	CreatedAt time.Time `json:"created_at" db:"created_at"`
+	CreateBy  *string   `json:"created_by" db:"created_by"`
+	UpdatedAt time.Time `json:"updated_at" db:"updated_at"`
+	UpdateBy  *string   `json:"updated_by" db:"updated_by"`
+	Data      string    `json:"data" db:"data"`
+	Locked    *int      `json:"isLocked" db:"locked"`
+}
+type DashboardData struct {
+	Description string                      `json:"description"`
+	Tags        []string                    `json:"tags"`
+	Name        string                      `json:"name"`
+	Layout      []Layout                    `json:"layout"`
+	Title       string                      `json:"title"`
+	Widgets     []map[string]interface{}    `json:"widgets"`
+	Variables   map[string]interface{}      `json:"variables"`
+	Version     string                      `json:"version"`
```

Comment:
There are `uuid` and `uploadedGrafana`  in the `data` object. Does it work fine if they miss in json?

Reply:
no it doesn't work that way  `unmarshal` won't pick data that is not specified in the type. added both as well